docs/scipp-neutron/from-mantid-to-scipp.ipynb

```diff
@@ -784,16 +855,42 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dspacing = sc.neutron.convert(tof, 'tof', 'd-spacing')"
+    "sc.neutron.convert(a_scipp, 'tof', 'd-spacing')"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Note that scipp has no equivalent to the `EMode` and `EFixed` settings of `ConvertUnits`.\n",
-    "Instead, this information is read from the input data, if available (note that currently only elastic scattering is supported)."
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**Note**\n",
+    "    \n",
+    "scipp has no equivalent to the `EMode` and `EFixed` settings of `ConvertUnits`.\n",
+    "Instead, this information is read from the input data, if available (note that currently only elastic scattering is supported).\n",
+    "    \n",
+    "</div>"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**Note**\n",
+    "\n",
+    "The `scipp.neutron module` also provides a `to_mantid` function, which has limited support for converting scipp data to Mantid workspaces. Because scipp offers a more flexible container than a Workspace, in particular MatrixWorkspace, it is not always possible to exactly convert all information to Mantid workspaces.\n",
```

Comment:
Can you split long lines at the end of each sentence?
The convention we adopted is one sentence per line.

Reply:
Indeed, will fix